ext/styledmap/eu.esdihumboldt.hale.doc.user.views.styledmap/html/wms_detail.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<html>
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
+<title>Map configuration via Web Map Service(WMS)</title>
+</head>
+<body>
+	<h1>WMS Configuration</h1>
+	<p>
+		The <i>WMS</i> provides functionality to add map using service URL. 
```

Comment:
replace with:

A Web Map Service (WMS) is a standard protocol for serving georeferenced map images which a map server generates using data from a GIS database. You can set up hale studo's Map view to use such a WMS.